e2e/cypress/integration/channel/user_to_admin_updates_manage_channel_members_modal_spec.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import users from '../../fixtures/users.json';
+
+const demoteToMember = (user) => {
+    const baseUrl = Cypress.config('baseUrl');
+    cy.externalRequest({user: users.sysadmin, method: 'put', baseUrl, path: `users/${user.id}/roles`, data: {roles: 'system_user'}});
+};
+
+const promoteToSysAdmin = (user) => {
+    const baseUrl = Cypress.config('baseUrl');
```

Comment:
Just like to chime in but baseUrl is actually no use here. It's already in the command itself - https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/support/task_commands.js#L41

Reply:
Removed. Thanks